src/main/java/jdbchomework/utils/ConnectionUtil.java

```diff
@@ -0,0 +1,43 @@
+package jdbchomework.utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+
+    private ConnectionUtil() {
+        throw new IllegalAccessError("Utility class");
+    }
+
+    private static Connection connection;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)